spec/features/form/validations_spec.rb

```diff
@@ -131,7 +131,7 @@ def fill_in_date(case_log_id, question, day, month, year, path)
         click_button("Save and continue")
         expect(page).to have_current_path("/logs/#{case_log.id}/net-income-value-check")
         expect(page).to have_content("Net income is outside the expected range based on the lead tenant’s working situation")
-        expect(page).to have_content("You told us the lead tenant’s working situation is: Full-time – 30 hours or more")
+        expect(page).to have_content("You told us the lead tenant’s working situation is: full-time – 30 hours or more")
```

Comment:
Should this refer to a translation string rather than text directly?

Reply:
Yep, will amend